postgres/docker-entrypoint-initdb.d/init-db.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
This does indeed create the tables, but couldn't we also create the database (if it didn't exist)?

Reply:
Sure!